### ISSUE TITLE
insufficient color contrast in tool box categories

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2271]
**Describe the bug**

the default categories do not have sufficient color contrast in the monaco toolbox

**To Reproduce**
1. Go to https://microbit.makecode.com
2. load https://makecode.microbit.org/_Lg7LJTi1uAck
3. See error (or use accessibility insights)

**Expected behavior**

colors should all have sufficent contrast (> 4.5:1). Some also have contrast issues against the monaco editor

**Current contrast ratios**

basic: 3.23
input: 4.42
music: 4.36
loops: 3.11
logic: 3.06
arrays: 3.65
text: 3.25

**Screenshots**

![Screen Shot 2019-07-17 at 3 49 17 PM](https://user-images.githubusercontent.com/5615930/61417102-dd2aff00-a8aa-11e9-909c-fc7f3966cdb8.png)

![Screen Shot 2019-07-17 at 3 49 27 PM](https://user-images.githubusercontent.com/5615930/61417101-dd2aff00-a8aa-11e9-8d5c-d1e26d617b1d.png)

**Additional context**

all listed in accessibility insights for web. high contrast mode makes all the text black so that might resolve this issue?

**Comment by BhuwanNaswa** [comment]
Labels
#A11yMAS, #HCL, #HCL-MakeCode-E2E, #MAS1.4.3, #Win10-Chrome, #Win10-Edge, #accessibility, #K4W, #closed
